src/Aspire.Hosting/Dcp/DcpExecutor.cs

```diff
@@ -54,6 +54,7 @@ internal sealed class DcpExecutor : IDcpExecutor
     private readonly ConcurrentDictionary<string, (CancellationTokenSource Cancellation, Task Task)> _logStreams = new();
     private DcpInfo? _dcpInfo;
     private Task? _resourceWatchTask;
+    private uint _stopped;
```

Comment:
nit: Could an `int` work? That's the idomatic default choice in .NET.

Reply:
Yep. Fixed.